src/pages/usePeoplePage.ts

```diff
@@ -0,0 +1,82 @@
+import { useEffect, useMemo, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { getPeople } from '../api';
+import { Person } from '../types';
+
+export const usePeoplePage = () => {
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [people, setPeople] = useState<Person[]>([]);
+  const [hasError, setHasError] = useState<boolean>(false);
+
+  useEffect(() => {
+    setIsLoading(true);
```

Comment:
it's already true, you don't have set it again

Reply:
okay done :P